src/modules/emote_menu/styles/menu.module.css

```diff
@@ -0,0 +1,75 @@
+.divider {
+  margin: 0px;
+}
+
+.header {
+  display: flex;
+  padding: 7.5px;
+}
+
+.content {
+  display: flex;
+  height: calc(360px - 52px);
```

Comment:
we should find a way to do this without calc. you can make use of margin and padding instead.